docs/reverse.rst

```diff
@@ -176,20 +176,22 @@ One such example is ``mailu/traefik-certdumper``, which has been adapted for use
     # !!! Also don’t forget to add "TRAEFIK_DOMAIN=[...]" to your .env!
       - DOMAIN=$TRAEFIK_DOMAIN
     volumes:
+      # Folder, which contains the acme.json
       - "/data/traefik:/traefik"
-      - "$ROOT/certs:/output"
+      # Folder, where cert.pem and key.pem will be written
+      - "/data/mailu/certs:/output"
 
 
-
-Assuming you have ``volume-mounted`` your ``acme.json`` put to ``/data/traefik`` on your host. The dumper will then write out ``/data/traefik/ssl/your.doma.in.crt``
-and ``/data/traefik/ssl/your.doma.in.key`` whenever ``acme.json`` is updated. Yay! Now let’s mount this to our ``front`` container like:
+Assuming you have ``volume-mounted`` your ``acme.json`` put to ``/data/traefik`` on your host. The dumper will then write out ``/data/mailu/certs/cert.pem`` and ``/data/mailu/certs/key.pem`` whenever ``acme.json`` is updated.
+Yay! Now let’s mount this to our ``front`` container like:
 
 .. code-block:: yaml
 
     volumes:
-      - "$ROOT/overrides/nginx:/overrides"
-      - /data/traefik/ssl/$TRAEFIK_DOMAIN.crt:/certs/cert.pem
-      - /data/traefik/ssl/$TRAEFIK_DOMAIN.key:/certs/key.pem
+      - /data/mailu/certs:/certs
+      - /data/mailu/certs:/certs
```

Comment:
same line twice?

Reply:
Good match. This is wrong. Probably a C&P issue. You can remove that doubled line, or I can update my PR.

Reply:
Fixed.